Sources/DateExtensions.swift

```diff
@@ -122,11 +122,12 @@ extension Date {
         } else if components.minute! >= 1 {
             components.minute == 1 ? (str = "minute") : (str = "minutes")
             return "\(components.minute!) \(str) ago"
-        } else if components.second == 0 {
-            return "Just now"
-        } else {
-            return "\(components.second!) seconds ago"
-        }
+		} else if components.second! >= 1 {
```

Comment:
I apologize, I don't know how will I fix it. Is it because I am using "Tabs" with Tab and Indentation width of four?
Thanks!